habitat-hitl/habitat_hitl/core/remote_client_state.py

```diff
@@ -156,12 +156,11 @@ def _update_input_state(self, client_states):
             input_json = (
                 client_state["input"] if "input" in client_state else None
             )
-            # TODO: Add mouse support
-            # mouse_json = (
-            #    client_state["mouse"] if "mouse" in client_state else None
-            # )
+            mouse_json = (
+                client_state["mouse"] if "mouse" in client_state else None
+            )
 
-            if input_json is not None:
+            if input_json != None:
```

Comment:
I think "is not" is the preferred pythonic way to check None still

Reply:
That's right!